tests/features/finish_by_hand.feature

```diff
@@ -6,22 +6,33 @@ Feature: Finish by hand
 # If I click continue on that page, I should be taken to the final page.
 
 # https://stackoverflow.com/questions/247135/using-xpath-to-search-text-containing-nbsp
-
+@justOneTest
```

Comment:
What does this do? Is this a language tag, or is this something else?

Reply:
Hi, @niharikasingh,

This line adds the ability to run just the tests in this file rather than the entire test suite.  A developer can run these tests by typing the following:

`npm run test -- --tags "@justOneTest"`

Let me update this to have  more descriptive name and then check in a final update.

Happy to simply remove this if you prefer.